lib/src/kvs/ds.rs

```diff
@@ -111,7 +111,7 @@ pub struct Datastore {
 	// Used only in some datastores, such as tikv.
 	versionstamp_oracle: Arc<Mutex<Oracle>>,
 	// Whether this datastore enables live query notifications to subscribers
-	notification_channel: Option<(Sender<Notification>, Receiver<Notification>)>,
+	notification_channel: OnceLock<(Sender<Notification>, Receiver<Notification>)>,
```

Comment:
Thanks @phughk. I'm loving the changes so far. Is there a reason for this particular change though? `OnceLock` is more expensive than just using an `Option`.

Reply:
Oh, I see. That diff threw me off then. It looks like it changed from `Option<(Sender<Notification>, Receiver<Notification>)>`.